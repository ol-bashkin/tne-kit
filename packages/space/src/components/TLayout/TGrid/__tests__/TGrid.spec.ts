import { mount } from "@vue/test-utils";
import TGrid from "../TGrid";

describe("TGrid.vue", () => {
  test("should have an t-grid class", () => {
    const wrapper = mount(TGrid, {
      propsData: {
        cols: "6",
      },
    });
    expect(wrapper.classes()).toContain("t-grid");
    expect(wrapper.html()).toMatchSnapshot();
  });
});
