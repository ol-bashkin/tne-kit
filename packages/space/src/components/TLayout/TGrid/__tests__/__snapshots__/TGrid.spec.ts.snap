// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TGrid.vue should have an t-grid class 1`] = `<div class="t-grid grid cols-6"></div>`;
